scss/partials/_menu.scss

```diff
@@ -480,17 +484,16 @@
 ------------------------------------------------------------*/
 .nav-overlay {
   animation: sweep-down 0.5s ease-in-out;
-  display: block;
   position: fixed;
   top: 0;
   left: 0;
   width: 100%;
   height: 100%;
   background-color: rgba(0, 0, 0, 0.5);
   z-index: 10;
+  display: none;
```

Comment:
Any reason why we're hiding the nav-overlay? In the menu.js file, the nav-overlay is created when any of the submenus are open the logic does not apply a style of display: block.

Reply:
Nice catch. Thank you. Must have been a merge issues when I was trying to update the branch.